.github/workflows/deploy-base.yml

```diff
@@ -0,0 +1,75 @@
+on:
+  workflow_call:
+    inputs:
+      env:
+        required: true
+        type: string
+    secrets:
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+      DOCKER_REPO:
+        required: true
+      SENTRY_AUTH_TOKEN:
+        required: true
+      SENTRY_ORG:
+        required: true
+      SENTRY_PROJECT:
+        required: true
+      SLACK_WEBHOOK:
+        required: true
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.env }}
+    env:
+      ECS_CLUSTER: skate
+      ECS_SERVICE: skate-${{ inputs.env }}
+      AWS_DEFAULT_REGION: us-east-1
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get version ids
+        id: version-ids
+        run: |
+          echo "sentry-release=${{github.ref}}_${{github.sha}}" | tr / - >> "$GITHUB_OUTPUT"
```

Comment:
Glides uses the tag as the sentry version when deploying from prod. Since we typically don't do deploys from tags, opting to use the ref + sha for all deploys.